public/css/dist/styles.css

```diff
@@ -474,6 +480,114 @@ pre {
   margin-bottom: 1rem;
   margin-right: 1rem;
 }
+/** TABLE OF CONTENT-Cards
+ — — — — — — — — — — — — — — — — — — — — — — — — — — — — — — /
+1 — Content Block
+  1.1 — Date
+  1.2 — Something Else
+
+/// END TABLE OF CONTENTS **/
+.content-block {
+  padding-right: 1.8rem;
+  position: relative;
+  width: calc(100vw - 2.8rem);
+}
+@media (min-width: 640px) {
+  .content-block {
+    padding-right: 7.5rem;
+    width: calc(100vw - 8.5rem);
+  }
+}
+.content-block__button {
+  display: none;
+}
+@media (min-width: 640px) {
+  .content-block__button {
+    display: inline-block;
+  }
+}
+.content-block__date, .content-block__image {
+  align-self: center;
+  grid-row: 1/2;
+}
+.content-block__date {
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  background-color: #003e51;
+  color: #ffffff;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  grid-column: 1/-1;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  margin-left: -1rem;
+  height: 9rem;
+  width: 9rem;
+}
+@media (min-width: 640px) {
+  .content-block__date {
+    margin-left: -2rem;
+  }
+}
+.content-block__date-month {
+  font-size: 1.4rem;
+  font-weight: 700;
+  letter-spacing: 0.3rem;
+  line-height: 1.6rem;
+}
+.content-block__date-day {
+  font-size: 4.2rem;
+  font-weight: 600;
+  line-height: 4.6rem;
+}
+.content-block__image {
+  grid-column: 1/3;
+  width: 100%;
+}
+.content-block__subtitle {
+  color: #477a7b;
+  margin-bottom: 1rem;
+}
+@media (min-width: 640px) {
+  .content-block__subtitle {
+    margin-bottom: 1.4rem;
+  }
+}
+.content-block__text {
+  margin-bottom: 2.4rem;
+}
+.content-block__title {
+  margin-bottom: 1rem;
+}
+@media (min-width: 640px) {
+  .content-block__title {
+    margin-bottom: 1.4rem;
+  }
+}
+.content-block__title > a {
+  text-decoration: none;
+  display: inline !important;
+}
```

Comment:
Not an elegant solution. Using the anchor class introduces `display: inline-block` which because of that the border is applied to the link as a block and not as an underline to the text. Need guidance.